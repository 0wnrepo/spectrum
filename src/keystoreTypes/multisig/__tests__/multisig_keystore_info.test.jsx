import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import MultisigKeystoreInfo from '../multisig_keystore_info';

const formData = {
  type: 'testType',
  addresses: [{
    name: 'test address 1',
    address: '0x12345',
    tokens: [{
      id: 1,
    }, {
      id: 2,
    }],
    networks: [{
      id: 1,
    }, {
      id: 2,
    }],
  }],
};

describe('<MultisigKeystoreInfo />', () => {
  test('renders correctly', () => {
    const component = shallow(
      <MultisigKeystoreInfo
        formData={formData}
      />,
    );

    expect(toJson(component)).toMatchSnapshot();
  });
});
