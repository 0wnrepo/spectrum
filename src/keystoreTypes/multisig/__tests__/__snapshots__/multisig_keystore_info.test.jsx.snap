// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MultisigKeystoreInfo /> renders correctly 1`] = `null`;
